docs/development/reviewing-patches.rst

```diff
@@ -34,6 +34,47 @@ These are small things that are not caught by the automated style checkers.
 * Does a variable need a better name?
 * Should this be a keyword argument?
 
+Testing branches remotely
```

Comment:
This implies that you're testing the branches on a remote machine. Could you reword?